### PR TITLE
add engine doc

### PR DESCRIPTION
**Pull request by Judahmeek**
Thanks to @noahprince22! From https://github.com/shakacode/react_on_rails/issues/562#issuecomment-257891002

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/614)

### CODE REVIEW

File: docs/additional-reading/rails-engine-integration.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
++ In your `<engine_name>.gemspec`:
+```
+s.add_dependency 'react_on_rails', '~> 6'
+```
++ In your `lib/<engine_name>.rb` (the entry point for your engine)
+```
+require "react_on_rails"
+```
++ In your `lib/tasks/<engine_name>_tasks.rake`:
+```
```
Comment (justin808) [inline docs/additional-reading/rails-engine-integration.md]: use this syntax:

``````
```ruby
``````

**Comment by justin808** [comment]
One comment. LGTM!

**Comment by Judahmeek** [comment]
Done.

**Comment by noahprince22** [comment]
Their might be an easier way to do this with `config.npm_build_production_command`. I'll test later and see if I can make this cleaner.

**Comment by justin808** [comment]
Thanks @Judahmeek!

@noahprince22 Please submit a new PR with updates!

**Pull request merged**